index.html

```diff
@@ -1,13 +1,56 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="./style/style.css">
-  <title>Killy's Portfolio</title>
-</head>
-<body>
-  
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Internal CSS File -->
+    <link rel="stylesheet" href="./style/style.css" />
+    <!-- External CSS File -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    />
+    <title>Killy's Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav id="logo" class="dark-blue">
+        <a href="index.html">Welcome</a>
+      </nav>
```

Comment:
- [ ] [Optional] It would be better semantically if you Remove the `<nav>` tag around your logo or include it beside the hamburger button in the same `<nav>` tag because you don't have multiple navbar it's just one in this design

Reply:
thanks for this!